docs/CHANGELOG.rst

```diff
@@ -27,6 +27,12 @@ Changed
   processes ``differentialexpression-edger`` and
   ``differentialexpression-deseq2``
 
+Fixed
```

Comment:
Please remove this. The changelog entries describing fixes of unreleased features are unnecessary, as the changelog text describing the introduction of this features is just a few lines above.